server/views/pages/recommendations/searchPpud.njk

```diff
@@ -30,42 +32,50 @@
                         Name
                     </dt>
                     <dd class="govuk-summary-list__value">
-                    {{ hyphenForBlank(recommendation.personOnProbation.fullName) }}
+                        {{ hyphenForBlank(recommendation.personOnProbation.fullName) }}
                     </dd>
                 </div>
                 <div class="govuk-summary-list__row">
                     <dt class="govuk-summary-list__key">
                         Date of birth
                     </dt>
                     <dd class="govuk-summary-list__value">
-                    {{ hyphenForBlank(formatDateTimeFromIsoString({isoDate: recommendation.personOnProbation.dateOfBirth})) }}
+                        {{ hyphenForBlank(formatDateTimeFromIsoString({isoDate: recommendation.personOnProbation.dateOfBirth})) }}
                     </dd>
                 </div>
                 <div class="govuk-summary-list__row">
                     <dt class="govuk-summary-list__key">
                         NOMIS ID
                     </dt>
                     <dd class="govuk-summary-list__value">
-                    {{ hyphenForBlank(recommendation.personOnProbation.nomsNumber) }}
+                        {{ hyphenForBlank(recommendation.personOnProbation.nomsNumber) }}
                     </dd>
                 </div>
                 <div class="govuk-summary-list__row">
                     <dt class="govuk-summary-list__key">
                         CRO
                     </dt>
                     <dd class="govuk-summary-list__value">
-                    {{ hyphenForBlank(recommendation.personOnProbation.croNumber) }}
+                        {% if recommendation.personOnProbation.croNumber %}
+                            {{ recommendation.personOnProbation.croNumber }}
+                            {% elif recommendation.prisonOffender.cro %}
+                            {{ recommendation.prisonOffender.cro }}
+                        {% else %}
+                            - <span class="govuk-visually-hidden">no value</span>
+                        {% endif %}
                     </dd>
                 </div>
             </dl>
             <div class='govuk-!-margin-top-6'>
                 <form novalidate method="post">
                     <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
                     <input type="hidden" name="crn" value="{{ recommendation.crn }}" />
-                    <input type="hidden" name="croNumber" value="{{ recommendation.personOnProbation.croNumber }}" />
+                    <input type="hidden" name="croNumber"
+                           value="{{ recommendation.personOnProbation.croNumber or recommendation.prisonOffender.cro }}" />
```

Comment:
I agree, you could set an `identifier` variable such as
```njk
{% set identifier = recommendation.personOnProbation.croNumber if recommendation.personOnProbation.croNumber else recommendation.prisonOffender.cro %}
```

this would allow you to do

```njk
{% if identifier %}
 {{ identifier }}
{% else %}
 - <span class="govuk-visually-hidden">no value</span>
{% endif %}
```

and 

```njk
<input type="hidden" name="croNumber" value="{{ identifier }}" />
```